database/migrations/2021_06_12_145807_add_multi_accounting_report.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the GNU Affero General Public License v3.0.
+// See the LICENCE file in the repository root for full licence text.
+
+use Illuminate\Database\Migrations\Migration;
+
+class AddMultiAccountingReport extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        DB::statement("ALTER TABLE osu_user_reports CHANGE reason reason ENUM(
+            'Insults',
+            'Spam',
+            'Cheating',
+            'UnwantedContent',
+            'Nonsense',
+            'MultiAccounting',
```

Comment:
Let's call this `MultipleAccounts` instead.

Also I'm not sure if you've checked whether it matters about order of adding into an enum in mysql, but I think putting it last would probably feel safer to me.

Reply:
Got it!